src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -102,11 +102,12 @@ class ElasticSearchDAO(servers:Seq[Authority], indexName: String) extends Search
     val searchStr =
     criteria.searchTerm match {
       case None | Some("") => ESQuery(new ESMatchAll).toJson.compactPrint
-      case Some(searchTerm:String) => ESQuery(new ESWildcard(searchTerm)).toJson.compactPrint
+      case Some(searchTerm:String) => ESQuery(new ESWildcard(searchTerm.toLowerCase)).toJson.compactPrint
```

Comment:
the standard analyzer uses by default a token filter 
"lowercase". we can define a custom analyzer if we want case sensitive matching